### PR TITLE
SS17 Final code changes. 

### PR DESCRIPTION
**Pull request by byteofprash**
Hi. This would be the Alpha release.

### CODE REVIEW

File: docs/usage/0_installation.rst [hunk 1]
```diff
@@ -41,7 +39,7 @@ Now you should be working in your virtual environment. We then will install all
 
 - Install OpenCV::
 
-    conda install -c menpo opencv3=3.2.0
+    conda install -c menpo opencv3=3.1.0
 
 - Install Tensorflow::
```
Comment (missxa) [inline docs/usage/0_installation.rst:44]: where are the instructions? add the link at least

File: src/FaceDetect.py [hunk 2]
```diff
@@ -10,38 +10,67 @@
 import dlib
 import cv2
 import RosMsgUtil
+import pickle
 
+def StartDetection(FrameQueue,RectQueue,FacepointQueue,SpeakerQueue):
```
Comment (missxa) [inline src/FaceDetect.py]: for all functions add doxygen format comments for the API reference on readthedocs (will be generated automatically)

- function description
- function's inputs with explanations and types
- function outputs & types

File: src/RecogniseFace.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    print("Time taken:",time.time()-start," Seconds")
+    vs = cv2.VideoCapture(0)
+    facesEmbeddings = []
+    if not vs:
+        sys.exit()
+    while True:
+        rects = RectsQueue.get()
+        FaceVal = []
+        if len(rects) >1:
+            print("MULTI FACEEEEEEEEEEEEEEEE")
```
Comment (missxa) [inline src/RecogniseFace.py]: hä?

**Review by missxa — CHANGES_REQUESTED** [review]

**Comment by missxa** [comment]
@byteofprash @Chris291 @bohare 

Function descriptions are missing almost everywhere

Ros messages are not sent! As discussed, for each face detected you send a message with:
- face id
- speaking (T/F)
- position in the frame for now (real world 3D coordinate in the future)
- name

**Comment by byteofprash** [comment]
@missxa Like I mentioned on telegram, the ids of the detected person is not consistent. It constantly shifts. So we need recognition to work to get the Ids working. We can of course send someone is talking, but to say who is talking, we would need id again. Position of the person is possible. Name of the person is again possible only after Recognition works. 

The recognition part doesn't work fully yet and we aren't going to demo recognition for the midterm demo.

**Pull request closed without merging**